libcalico-go/lib/health/health.go

```diff
@@ -177,19 +182,50 @@ func (aggregator *HealthAggregator) Summary() *HealthReport {
 	// In the absence of any reporters, default to indicating that we are both live and ready.
 	summary := &HealthReport{Live: true, Ready: true}
 
+	// Prepare a table to report detail.
+	var buf bytes.Buffer
+	table := tablewriter.NewWriter(&buf)
+	table.SetHeader([]string{"COMPONENT", "TIMEOUT", "LIVENESS", "READINESS", "DETAIL"})
```

Comment:
Does this table format decently in `kubectl describe pod` output? My assumption based on the rest of this is that writing to the response body will correctly show up in `kubectl describe`, which is awesome, but I'm wondering if it's human-readable in the describe output